quiz/web/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:19.7.0-slim as builder
+
+WORKDIR /opt/app
+
+COPY ./package.json ./package-lock.json ./
+RUN npm ci
+
+COPY ./public ./public
+COPY ./nginx ./nginx
+COPY ./src ./src
```

Comment:
Zamiast kopiować w ten sposób, zrobiłbym kopie wszystkiego i .dockerignorem wylączyłbym to czego nie potrzebujemy, jeżeli będziemy chcieć coś dodać do conatiner to trzeba będzie zmieniąc dockerfile